substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2573,17 +2588,40 @@ pub mod pallet {
 
 		/// Chill on behalf of the pool.
 		///
-		/// The dispatch origin of this call must be signed by the pool nominator or the pool
+		/// The dispatch origin of this call can be signed by the pool nominator or the pool
 		/// root role, same as [`Pallet::nominate`].
 		///
+		/// Under certain conditions, this call can be dispatched permissionlessly (i.e. by any
+		/// account).
+		///
+		/// # Conditions for a permissionless dispatch
+		///
+		/// * When pool depositor has less than `MinNominatorBond` staked, otherwise  pool members
+		///   are unable to unbond.
+		///
+		/// # Conditions for permissioned dispatch
+		///
+		/// * The caller has a nominator or root role of the pool.
+		///
```

Comment:
```suggestion
		/// Under certain conditions, this call can be dispatched permissionlessly (i.e. by any
		/// account).
		///
		/// # Conditions for a permissionless dispatch:
		/// * When pool depositor has less than `MinNominatorBond` staked, otherwise  pool members
		///   are unable to unbond.
		///
		/// # Conditions for permissioned dispatch:
		/// * The caller has a nominator or root role of the pool.
```